SU-HSDO/suhumsci
# SHS-5661: Remove legacy fields

# READY FOR REVIEW

## Summary
- Removes legacy field Hero Overlay Color (`field_hs_hero_overlay_color`) from the Banner image with full overlay and text paragraph.
- Tugboat testing site: https://pr1577-tvov9lhfq1uslcrulxek4dlu6rygpis7.tugboatqa.com

## Need Review By (Date)
7/22/2024

## Urgency
low

## Steps to Test
1. Login to the site, and adda new Flexible Page
2. Click "Add Component" under the "Main Region"
3. Add a component of type "Banner image(s) with text box"
4. Verify that the field "Overlay Color" no longer exists.
5. As an additional check, visit the paragraph settings at `admin/structure/paragraphs_type/hs_hero_image/fields` and verify that there is no longer an "Overlay Color" field.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207828255718802
  - https://app.asana.com/0/0/1207779507223969
